server/command/command.go

```diff
@@ -65,29 +67,37 @@ func GetSlashCommand() *model.Command {
 }
 
 func addSubCommands(splunk *model.AutocompleteData) {
-	splunk.AddCommand(createAlertCommand())
+	splunk.AddCommand(addSubscribeCommand())
 	splunk.AddCommand(createAuthCommand())
 	splunk.AddCommand(createHelpCommand())
 	splunk.AddCommand(createlogCommand())
 }
 
-func createAlertCommand() *model.AutocompleteData {
-	alert := model.NewAutocompleteData(
-		"alert", "--subscribe", "subscribe to alert")
+func addSubscribeCommand() *model.AutocompleteData {
+	subscribe := model.NewAutocompleteData(
+		"alert", "[command]", "Available commands: subscribe, list, delete")
 
-	flag := []model.AutocompleteListItem{
-		{HelpText: "Subscribe", Item: "--subscribe"},
-	}
-	alert.AddStaticListArgument("Subscribe to alert", true, flag)
-	return alert
+	addAlert := model.NewAutocompleteData(
+		"subscribe", "", "Subscribe to an alert")
+	subscribe.AddCommand(addAlert)
+	deleteAlert := model.NewAutocompleteData(
+		"delete", "", "Remove an alert")
+	deleteAlert.AddTextArgument("AlertId to remove", "[alertid]", "")
```

Comment:
Suggestion to add newlines between the different commands

```suggestion
	addAlert := model.NewAutocompleteData(
		"subscribe", "", "Subscribe to an alert")
	subscribe.AddCommand(addAlert)

	deleteAlert := model.NewAutocompleteData(
		"delete", "", "Remove an alert")
	deleteAlert.AddTextArgument("AlertId to remove", "[alertid]", "")
```